canvas_data_warehouse/views.py

```diff
@@ -8,14 +8,18 @@
 
 class CanvasDataViewSet(viewsets.ViewSet):
     '''create API views for getting graph data'''
-    @list_route(name='get-canvas-data')
+    @list_route(url_path='data')
```

Comment:
Since this class is inherited from `viewsets.ViewSet`, you can just override `def get(self, request)`, then you don't have to add extra decorators.